purchase_create_related/wizards/purchase_order_secondary_linker.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class PurchaseOrderSecondaryLinker(models.TransientModel):
+    _name = "purchase.order.secondary.linker"
+    _description = "Wizard to link purchases to secondary purchases"
+
+    operation_type = fields.Selection(
+        selection=[
+            ("create", "Create Quotation Purchase"),
+            ("link", "Link to Quotation Purchase"),
+        ],
+        required=True,
```

Comment:
no default is indicated?

Reply:
I think it is not neccesary, users will have to read the field and chose one